internal/daemon/cluster/handlers/worker_service.go

```diff
@@ -201,6 +203,38 @@ func (ws *workerServiceServer) Status(ctx context.Context, req *pbs.StatusReques
 	return ret, nil
 }
 
+func (ws *workerServiceServer) HcpbWorkers(ctx context.Context, req *pbs.HcpbWorkersRequest) (*pbs.HcpbWorkersResponse, error) {
```

Comment:
Can you add a function comment explaining that this is used to find HCPB workers by looking up KMS workers?
```suggestion
func (ws *workerServiceServer) LookupHcpbWorkers(ctx context.Context, req *pbs.HcpbWorkersRequest) (*pbs.HcpbWorkersResponse, error) {
```

Reply:
Will do